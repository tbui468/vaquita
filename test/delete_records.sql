if exists drop database sol;
create database sol;
open sol;

create table planets (
                        id int key,
                        name string, 
                        mass float, 
                        number_of_moons int,
                        surface_pressure float,
                        ring_system bool
                    );

insert into planets (
                        id,
                        name, 
                        mass, 
                        number_of_moons, 
                        surface_pressure,
                        ring_system
                    ) 

values
        (1, "Mercury", 0.330, 0, 0.0, false),
        (2, "Venus", 4.87, 0, 92.0, false),
        (3, "Earth", 5.97, 1, 1.0, false);

delete from planets where id >= 2;
select * from planets;
close sol;
drop database sol;
exit;
