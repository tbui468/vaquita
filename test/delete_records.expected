created database sol
opened database sol
created table planets
inserted 8 record(s) into planets
4, Mars, 0.642000, 2, 0.010000, false, 
closed database sol
dropped database sol
